.github/workflow-template/template.yml

```diff
@@ -16,8 +16,6 @@ env:
   AWS_ROLE_TO_ASSUME: arn:aws:iam::639303875316:role/Create-IAM-Role-for-Configure-AWS-Credentials-Role-1NF1LWROB80QG
 
 permissions:
```

Comment:
```suggestion
```

Reply:
I leave it here on purpose, to hint that `permissions` key will be overridden by other yaml files 🤣